app/controllers/admin/cases_controller.rb

```diff
@@ -182,4 +188,13 @@ def correspondence_types_map
   def case_and_type
     "case_#{@correspondence_type_abbreviation}".to_sym
   end
+
+  def set_correspondence_type
+    if params[:correspondence_type].present?
+      @correspondence_type = CorrespondenceType.find_by_abbreviation(
+        params[:correspondence_type].upcase
```

Comment:
I think `find_by_<attribute_name>(<attribute_value>)` is deprecated in favour of `find_by(<attribute_name>: <attribute_value>)`

Reply:
I agree that I'd _prefer_ to use `find_by`, and I'll change this (not sure how it got there, might be ⌘C⌘V), but as it turns out `find_by_<attribute>` isn't deprecated! I know, I thought it was too!